test/hisi_zip_test/test_sva_perf.c

```diff
@@ -1085,6 +1089,26 @@ static int run_self_test(void)
 			opts.sync_mode = 1; 	opts.poll_num = 8;
 			opts.block_size = 8192; opts.total_len = 8192 * 10;
 			break;
+		case 5:
+			opts.sync_mode = 0;
+			opts.block_size = 1024; opts.total_len = 8192 * 10;
+			break;
+		case 6:
+			opts.sync_mode = 1; 	opts.poll_num = 1;
+			opts.block_size = 1024; opts.total_len = 8192 * 10;
+			break;
+		case 7:
+			opts.sync_mode = 1; 	opts.poll_num = 2;
+			opts.block_size = 1024; opts.total_len = 8192 * 10;
+			break;
+		case 8:
+			opts.sync_mode = 1;	opts.poll_num = 4;
+			opts.block_size = 1024;	opts.total_len = 8192 * 10;
+			break;
+		case 9:
+			opts.sync_mode = 1; 	opts.poll_num = 8;
+			opts.block_size = 1024; opts.total_len = 8192 * 10;
+			break;
```

Comment:
better to make them dynamically, if you go through current block test, you will find it already support any size block size :)